llvm/test/CodeGen/AMDGPU/insert-delay-alu-bug.ll

```diff
@@ -3,6 +3,18 @@
 
 declare i32 @llvm.amdgcn.workitem.id.x()
 
+define <2 x i64> @f1() #0 {
```

Comment:
Unrelated function appeared?

Reply:
There were some function order changes in some tests. This function is the same one as the function that is removed a couple of lines below.